Gaijinizer patch/Map172.txt

```diff
@@ -1163,11 +1182,10 @@ over twenty years ...! It's not so difficult.#MTLed
 
 まさにギリギリだったってわけだ……！
 > CONTEXT: Map172/events/2/pages/1/226/Dialogue/kazuya-3
-If you are in a coma and if "the evil spirit"
-dives into the bottom of your consciousness
-No matter what kind of treatment you can do, you
-can not take your hands off ...
-It was just just barely ......!#MTLed
+If this "malice" has used the coma has an
+opportunity to bury itself deep within his
+consciousness, you have no hope of removing it.
+You were so close, too...!#RonIffyWording
 > END STRING
```

Comment:
I think this gets at what he's saying better and is much clearer:
>_**Once he went into a coma and the "malice" was
buried deep into his consciousness, you had
no hope of removing it.**_